cmd/cape/cmd/token_test.go

```diff
@@ -1,19 +1,104 @@
 package cmd
 
 import (
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/json"
 	"os"
 	"path/filepath"
 	"testing"
+	"time"
+
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	"github.com/lestrrat-go/jwx/v2/jwt"
+	"github.com/spf13/viper"
 )
 
+// `cape token` uses the token subject from the currently logged in cape user.
+// this helper creates a dummy auth file for that purpose.
+func beforeOnce() error {
+	// We cannot easily validate the JWKS in test as the test access token is self generated.
+	getAccessTokenVerifyAndParse = func() (jwt.Token, error) {
+		tokenResponse, err := getTokenResponse()
+		if err != nil {
+			return nil, err
+		}
+		return jwt.Parse([]byte(tokenResponse.AccessToken), jwt.WithVerify(false))
+	}
+
+	localConfigDir, err := os.MkdirTemp("", "config")
```

Comment:
You're also supposed to

```go
defer os.RemoveAll(localConfigDir)
```

Reply:
:cherries:

Reply:
@bendecoste see updated to support removeall